docs/content/tutorials/firewall.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: doc_page
+---
+
+What to Do When You Have a Firewall
+-----------------------------------
+When you are behind a firewall, the Maven Druid dependencies will not be accessible, as well as the IRC wikipedia channels that feed realtime data into Druid. To workaround those two challenges, you will need to:
+
+1. make the Maven Druid dependencies available offline
```

Comment:
It doesn't make sense to present these as a numbered list. Like, you _have to_ do 1 if you can't reach the internet. But 2 and 3 are more of a choice. You just have to do one of the other.